packages/augur-ui/src/modules/types.ts

```diff
@@ -815,3 +815,23 @@ export interface Categories {
   tertiary: string;
 }
 
+export interface Location {
```

Comment:
I'm pretty sure there are types available for these already for the webpack shims.